lib/custom-resources/delete-default-sg-rules.ts

```diff
@@ -0,0 +1,86 @@
+import { EC2, SecurityGroupRule } from "@aws-sdk/client-ec2";
+import type { OnEventRequest, OnEventResponse } from "aws-cdk-lib/custom-resources/lib/provider-framework/types";
+
+const ec2 = new EC2({ useFipsEndpoint: true });
+
+/**
+ * Handle CloudFormation Custom Resource events using the CDK Custom Resource Provider Framework.
+ *
+ * This will, given a VPC Endpoint ID, determine the full list of IP addresses associated with
```

Comment:
Copypasta?

Reply:
Fixed in bab938fd76512a81384c1c2a49727e8cc2e75a35 along with other docs being added.